### ISSUE TITLE
Names for sticks does not correspond to source

### ISSUE THREAD

**Issue by sande005** [FrSkyRC/ETHOS-Feedback-Community#4267]
In the Sticks setting, the names for the sticks do no correspond to the actual source - throttle/elevator labels transposed for mode 2, etc.
Ver 1.5.11, upgrade from installed 1.0.9 on new just out of the box X10S-Express.  "Throttle warning", etc. corresponds correctly to the selected Mode.  First noticed in very first airplane setup that the name for the source was wrong.

**Comment by Wolepo** [comment]
The names of the stick functions are freely selectable, so simply rename them.

**Comment by bsongis-frsky** [comment]
I am not sure I understand what you mean.
Stick Modes should work flawlessly since long
Please paste your radio.bin, perhaps you have changed the sticks names manually?

**Comment by sande005** [comment]
Received a new X10S-E from Horus.  Had ETHOS 1.0.9 installed from the factory.  Turned it on, and it created a model via the Wizard.  Did not even look at what was there.  Updated it all to 1.5.11 before doing anything else.  Created a test model via the Wizard (there were no other models), and noticed that the sources in the mixer were not lining up with the outputs, but they did function correctly.  Went to System/Sticks, and for mode 2, T was labeled Elevator, and E was labeled Throttle.  Same kind of mismatch for Mode 1.  Went back to Mode 2, and labels continued to mismatch.  So just the naming of the labels were not corresponding, modes worked correctly.
Interestingly, I flashed it over to Edge TX like my previous X10S-E.  Then later thought I should give Ethos a fair period of use before deciding.  This time did a complete install (blank SD card and all) direct to 1.5.11, and all labeling was then correct.
